.cloudbuild/scripts/internal/artifacts/storage.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package artifacts
+
+import (
+	"context"
+	"io"
+
+	"cloud.google.com/go/storage"
+)
+
+// bucket presents a minimal interface to a storage bucket, allowing us to
+// mock out GCB for testing
+type objectHandle interface {
```

Comment:
Are the interfaces here widely used? Could we move them closer to the point where they are used?

Reply:
moved